src/methods/fetch-store-applications.js

```diff
@@ -16,10 +21,31 @@ ecomApps.fetchStoreApplications()
 
  */
 
-export default (self, appId) => {
-  let url = '/applications.json'
-  if (appId) {
-    url += `?app_id=${appId}`
+export default (self, meta = {}) => {
+  const { params, fields, limit, offset, sort } = meta
+  let url = '/applications.json?'
+  if (params) {
+    for (const key in params) {
+      switch (typeof params[key]) {
+        case 'string':
+        case 'number':
+          url += `${key}=${params[key]}&`
+      }
+    }
+  }
+  if (limit) {
+    url += `&limit=${limit}`
+  }
+  if (offset) {
+    url += `&offset=${offset}`
+  }
+  if (sort) {
+    url += `&sort=${sort}`
+  }
+  if (fields && Array.isArray(fields) && fields.length) {
+    url += `&fields=${fields.join()}`
+  } else if (fields && typeof fields === 'string') {
+    url += fields
```

Comment:
Não tem mais `join` porque não é um array, mas ainda é preciso adicionar corretamente à query string:

```suggestion
    url += `&fields=${fields}`
```